assets/src/dashboard/app/views/editorSettings/index.js

```diff
@@ -240,6 +248,20 @@ function EditorSettings() {
         return setMediaError(errorText);
       }
 
+      if (!allFileTypeSupported) {
+        const errorText =
+          files.length === 1
+            ? __(
+                'Sorry, this file type is not supported. Please upload another file. ',
+                'web-stories'
+              )
+            : __(
+                'Sorry, one or more of these files are not a supported file type. Make sure your logos are the correct file type.',
```

Comment:
```suggestion
                'Sorry, one or more of these files are of an unsupported file type. Make sure your logos are of the correct file type.',
```

Reply:
@swissspidy @spacedmonkey : As a novice end user, I might benefit from a recommendation of which file types are supported in this error message. Either an explicit list, or if the image types might change in future, a link to the current list supported image types.

Suggestion: 
'Sorry, one or more of these files are of an unsupported file type. Only jpg, png, and static gifs are supported for publisher logos'

Reply:
Wording as been updated with 
`Sorry, this file type is not supported. Only jpg, png, and static gifs are supported for publisher logos.`
`Sorry, one or more of these files are of an unsupported file type. Only jpg, png, and static gifs are supported for publisher logos.`